addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -638,6 +647,13 @@ export class LinkPlugin extends Plugin {
                 if (closestLinkElement !== this.linkInDocument) {
                     this.openLinkTools(closestLinkElement);
                 }
+            } else if (
```

Comment:
isEditableLink(el) {
this.delegateTo("blabla", el )
}

"blabl": (el ) => {
(el.getAttribute("role") === "menuitem" ||
                    el.classList.contains("nav-link"))